### ISSUE TITLE
Add immutable ids for subsection data

### ISSUE THREAD

**Issue by sarabveer** [shabados/database#1338]
Currently subsections are useless as subsection names are not unique

**Comment by bhajneet** [comment]
How does this issue affect the average user?

**Comment by Sarabveer** [comment]
Not sure about the average user, but this is required for v5. Otherwise subsections are useless as many share the same name.

**Comment by Sarabveer** [comment]
@saihaj Instead of having something like `"subsection": "Jap",` in the json, it would be an immutable id, like `"subsection": "HJ6",`

You can see all the current defined subsections here in https://github.com/ShabadOS/database/blob/dev/data/compositions.json. Many share the same English name, they aren't unique, so we need immutable ids.

Can probably touch up with @Harjot1Singh on the schema.

**Comment by saihaj** [comment]
* `section`
    * `id` -> 3 char immutable 
     * `order_id` -> increasing integer
* `subsection`
     * `id` -> 3 char immutable 
     * `order_id` -> increasing integer

[View in Slack](https://shabados.slack.com/archives/C8LH4UF5H/p1593352303047400)

**Comment by Harjot1Singh** [comment]
Yep you've got it, see lines or shabads as an example, they both have `order_id` and `id`.

**Comment by Harjot1Singh** [comment]
@Sarabveer Would you not prefer to have `subsection: "Jap"`?

**Comment by Sarabveer** [comment]
@Harjot1Singh subsection names arent unique, which is why this issue was opened.

**Comment by Sarabveer** [comment]
@Harjot1Singh subsection names arent unique, which is why this issue was opened.

**Comment by Sarabveer** [comment]
Section doesn't need immutable IDs as up to this point, they are unique. However, within a section, there can be multiple subsections sharing the same name. So immutable IDs for subsections are needed if we want to use them, for example, as bookmarks feature (in a Baani) for a Gutka app.
